Neos.Flow/Classes/Security/Authentication/Provider/PersistedUsernamePasswordProvider.php

```diff
@@ -51,6 +52,12 @@ class PersistedUsernamePasswordProvider extends AbstractProvider
      */
     protected $persistenceManager;
 
+    /**
+     * @var PrecomposedHashProvider
+     * @Flow\Inject
```

Comment:
Make this `(lazy=false)`, else when we fall into the case the instanciation of the HashProvider will change the runtime again ;)
Optimally also directly document in the doc block why it's non-lazy, so any future "optimization attempts" get reminded of the impact